cmd/osv-scanner/main.go

```diff
@@ -20,10 +20,10 @@ var (
 )
 
 func run(args []string, stdout, stderr io.Writer) int {
-	var r *reporter.Reporter
+	var r reporter.Reporter
 
 	cli.VersionPrinter = func(ctx *cli.Context) {
-		r = reporter.NewReporter(ctx.App.Writer, ctx.App.ErrWriter, "")
+		r = reporter.NewTableReporter(stdout, stderr, false)
```

Comment:
This should not have been changed because it causes tests to be flakey - see #166